usage/index.adoc

```diff
@@ -16,6 +16,7 @@ To do that there are two separate functionalities:
   wish.
 
 <<Error reporting>>::
+  *Notice - As of April 28, 2020, JBoss Tools 4.15 will suspend AERI feature operations. Many thanks to everyone who participated!*
```

Comment:
Add an empty line to visually separate the new and old sections

Reply:
added breaking line.